src/winter_onboarding_2021/fleet_management_service/handlers/cab.clj

```diff
@@ -0,0 +1,47 @@
+(ns winter-onboarding-2021.fleet-management-service.handlers.cab
+  (:require [ring.util.response :as response]
+            [clojure.spec.alpha :as s]
+            [winter-onboarding-2021.fleet-management-service.models.cab :as cab]
+            [winter-onboarding-2021.fleet-management-service.views.layout :as layout]
+            [winter-onboarding-2021.fleet-management-service.views.cab :as cab-views]))
+
+(s/def :cab/name string?)
```

Comment:
Maybe the specs can move to the model namespace. If someone is looking to understand the structure of a cab, they just need to look at the models in that case.